docs/SQL/development/visual-editor/variant-schema.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Variant schema
+id: variant-schema
+description: About variant schema
+sidebar_position: 2
+tags:
+  - concept
+  - variant
+  - sql
+---
+
+You can use Prophecy to parse your semi-structured data types, and convert them into flat, structured formats to make them easier to understand and use for analytics. This is available for when you want to determine the variant schema of your Snowflake arrays.
```

Comment:
We can also say `Snowflake Object` apart from Arrays. In Flatten Schema gem, if user clicks "Add All" Instead of each column level then the all nested leaf level values of an Object would be visible to user as columns.

Reply:
![Uploading image.png…]()

Reply:
Done.